core/src/avm2/api_version.rs

```diff
@@ -0,0 +1,63 @@
+use gc_arena::Collect;
+
+// Based on https://github.com/adobe/avmplus/blob/master/core/api-versions.h
+// FIXME - if we ever add in AIR emulation, then implement the 'version series' logic.
```

Comment:
If you consider adding that, would Ruffle devs have to support Adobe AIR only features too?

Reply:
It's the other way around - if we want to implement any AIR-only APIS, then we may need to add full support for AIR api-versioning.